authors/apps/authentication/serializers.py

```diff
@@ -15,6 +15,7 @@ class RegistrationSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
+    token = serializers.CharField(max_length=225, read_only=True)
```

Comment:
Why does the `RegistrationSerializer` have a token?

Reply:
Thanks for the feedback @Karuhanga. but adding the token field in the registration serializer was a requirement in my task's description where I was asked to return a token to a user that has been successfully registered.
Thanks for the feedback again.

Reply:
Just seen the description on `PT`. True. Perharps @malep2007 should clarify on why this is necessary.

Reply:
The way I think about it, a user will not want to go back and sign in after initial registration. They will want to be signed in automatically after successful registration - of course factoring in the email verification too. So it makes sense to return an access token after this.